plugins/entando-plugin-jacms/src/main/java/org/entando/entando/plugins/jacms/aps/system/services/resource/ResourcesService.java

```diff
@@ -149,13 +165,25 @@ public void deleteAsset(String resourceId) {
         }
     }
 
-    public AssetDto editAsset(String resourceId, MultipartFile file, String description, List<String> categories) {
+    private ResourceInterface loadResource(String resourceId) {
```

Comment:
Method should be called getAsset.
Also, resourceInterface should never be exposed, even internally. Use AssetDto instead.